test/connection.test.ts

```diff
@@ -513,7 +513,7 @@ describe('Connection', function () {
       })
     })
 
-    it('should throw an error if the exchange rate is smaller than 1/1000000000', async function () {
+    it.skip('should throw an error if the exchange rate is smaller than 1/1000000000', async function () {
```

Comment:
Added skip to this, @emschwartz why do we want to throw and error if the exchange rate is that small. If it has enough precision then I think we should consider it to be valid.

Reply:
I think that's in there because it doesn't send a big enough test packet to find a rate that small

Reply:
Hmm... if we send `const testPacketAmounts = [1, 1000, 1000000, 1000000000]` then it should be able to discover that exchange rate, they both have 9 zeros. 

I think it was previously failing since the max digits would be one on the prepareAmount when you send the 1000000000 so it would fail to establish. With this change it would find the 1000000000 -> 1 rate.

Reply:
True, then you should take that test out instead of skipping it